src/guide/loops.md

```diff
@@ -23,7 +23,10 @@ Loops can skip to the next iteration using the `continue` keyword. For example,
 let mut i = 0
 
 while (i < 10) {
```

Comment:
I think it might be better if we use a for loop here.
```rs
for (let mut i = 0; i <= 10; i += 1) {
  if (i % 2 == 0) continue;
  print(i)
}
```

Reply:
Therefore this code snippet should be moved into `for` loop section.

So feel free to close my PR and make your own changes, or make changes here.

I do not feel experienced enough to change the structure of this documentation.